elastic/kibana
# [maps] fix no geometry column found when editing ES|QL layers

Resolves https://github.com/elastic/kibana/issues/176227

PR updates maps ES|QL code to handle response shape when `drop_null_values` is used. PR adds an integration test that will fail if the ES|QL response shape changes.

#### Test steps
1. install sample web logs
2. create new map, add `ES|QL` layer
3. Edit query, changing limit from default of "10,000" to "1,000". Click "run". Verify results are displayed
4. Set query that returns no results like, "bytes > 100000000". Verify layer displays no results.
